bin/node-template/docker-compose.yml

```
@@ -3,7 +3,7 @@ version: "3.2"
 services:
   dev:
     container_name: node-template
-    image: paritytech/ci-linux:974ba3ac-20201006
+    image: paritytech/ci-linux:staging
```

```suggestion
    image: paritytech/ci-linux:production
```
`paritytech/ci-linux:production` is used in ci pipeline, `staging` can have a newer, not yet supported version of rust